Marlin/src/HAL/STM32F1/tft/tft_fsmc.cpp

```diff
@@ -89,6 +89,16 @@ void TFT_FSMC::Init() {
   uint8_t cs = FSMC_CS_PIN, rs = FSMC_RS_PIN;
   uint32_t controllerAddress;
 
+  #if PIN_EXISTS(TFT_BACKLIGHT)
+    OUT_WRITE(TFT_BACKLIGHT_PIN, DISABLED(DELAYED_BACKLIGHT_INIT));
+  #endif
```

Comment:
If this pin is enabled here does it also need to be enabled again below (after `TFT_RESET`)?